smoke-tests/springboot/src/main/java/io/opentelemetry/smoketest/springboot/controller/WebController.java

```diff
@@ -17,13 +17,18 @@
 package io.opentelemetry.smoketest.springboot.controller;
 
 import io.opentelemetry.extensions.auto.annotations.WithSpan;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class WebController {
+  private static final Logger LOGGER = LoggerFactory.getLogger(WebController.class);
+
   @RequestMapping("/greeting")
   public String greeting() {
+    LOGGER.info("HTTP request received");
```

Comment:
Tests fail because they don't build this application as part of the test. If you are sure that test work, you may submit two separate PRs: one for this app and another for the actual test

Reply:
I've tested it locally and it worked -- split into two PRs.
Thanks!